marlowe-dashboard-client/src/Contract/View.purs

```diff
@@ -152,9 +153,13 @@ actionConfirmationCard assets state namedAction =
         ]
       _ -> [ transactionFeeItem false ]
 
+    -- TODO: when we allow custom currency other than ada, we won't be able to add the deposit amount
```

Comment:
We should take this into account when Russ provides the new Payments design, in which we probably need to put different costs (in different currency) and multiple payments